runtime/src/genesis_utils.rs

```diff
@@ -200,6 +201,20 @@ pub fn activate_all_features(genesis_config: &mut GenesisConfig) {
     }
 }
 
+pub fn deactivate_features(genesis_config: &mut GenesisConfig, features_to_skip: &Vec<Pubkey>) {
```

Comment:
nit: Since the function name is now `deactivate_features`, can you change the parameter name to `features_to_deactivate` ?